src/components/ComponentRelation/details-page/ComponentNudges.tsx

```diff
@@ -63,7 +64,7 @@ const ComponentNudgesSVG: React.FC<ComponentNudgesSVGprops> = ({
         {relatedComponents.map((comp, i) => (
           <li key={i} data-test="nudges-connector">
             <Link
-              to={`/workspaces/${workspace}/applications/${comp.spec?.application}/components/${comp.metadata.name}`}
+              to={`/workspaces/${namespace}/applications/${comp.spec?.application}/components/${comp.metadata.name}`}
```

Comment:
Also should not be this like:

to={`${APPLICATION_DETAILS_PATH.createPath({ workspaceName: namespace, applicationName: comp?.spec?.application, componentName: comp.metadata.name})}/`} or something similar @JoaoPedroPP @CryptoRodeo